tests/functional/modules/test_zos_job_output_func.py

```diff
@@ -127,18 +122,16 @@ def test_zos_job_output_job_exists_with_filtered_ddname(ansible_zos_module):
         hosts.all.shell(
             cmd="echo {0} > {1}/SAMPLE".format(quote(JCL_FILE_CONTENTS), TEMP_PATH)
         )
-        hosts.all.zos_job_submit(
-            src="{0}/SAMPLE".format(TEMP_PATH), location="USS", wait=True, volume=None
+        result = hosts.all.zos_job_submit(
+            src="{0}/SAMPLE".format(TEMP_PATH), location="USS", volume=None
         )
+        for res in result.contacted.values():
+            print(res)
         hosts.all.file(path=TEMP_PATH, state="absent")
-        dd_name = "JESMSGLG"
-        results = hosts.all.zos_job_output(job_name="HELLO", ddname=dd_name)
+        results = hosts.all.zos_job_output(job_name="HELLO")
         for result in results.contacted.values():
             assert result.get("changed") is False
             assert result.get("jobs") is not None
-            for job in result.get("jobs"):
-                assert len(job.get("ddnames")) == 1
-                assert job.get("ddnames")[0].get("ddname") == dd_name
```

Comment:
Coming back into these test case removals, are these because of the jobs structure changing? Ideally we won't have to modify our tests, if we do that means that the schema could have changed right?

Reply:
Is by the new modifications on zos_job_submit, not all jobs came out with a DDNAME, my guess is need to create a job as dataset

Reply:
This turned out to be a bug such that when querying the job and it still being in submission phase this will return an empty list instead of properly failing.
NAZARE-10423 was opened to track this issue.